spectrum/test_journal.py

```diff
@@ -66,6 +66,19 @@ def test_rss_feeds():
 def test_login():
     session = input.JOURNAL.session()
     session.login()
+    # no pagination needed so far
+    profiles = checks.API.profiles()['items']
```

Comment:
To changed once https://github.com/elifesciences/journal/pull/746 is merged I guess?

Reply:
Once we have a profile page we can look at that and perhaps extract the id of the profile from there.